lib/state/action/pokemon_actions.dart

```diff
@@ -32,3 +35,9 @@ class GetPokemonDetailsAction extends ReduxAction<AppState> {
     return state.copyWith(currentPokemonDetails: currentPokemonDetails);
   }
 }
+
+// Dispose current details on the store
```

Comment:
this is a comment, documentation should be using `///` and maybe you can improve its documentation details based on the action's updated name 🙂